### ISSUE TITLE
Polling station information not correctly displayed

### ISSUE THREAD

**Issue by RaduCStefanescu** [code4romania/monitorizare-vot-ong#174]
The polling station information is not correctly displayed on the frontend:
<img width="532" alt="Screenshot 2019-11-05 at 09 42 56" src="https://user-images.githubusercontent.com/15039873/68188008-d7b80880-ffb0-11e9-9359-1e4553e129f8.png">
```
{"lastModified":"2019-11-05T07:28:51.293","urbanArea":true,"observerLeaveTime":null,"observerArrivalTime":"2019-11-05T12:29:05","isPollingStationPresidentFemale":true}
```

**Comment by idormenco** [comment]
on it

**Comment by RaduCStefanescu** [comment]
Fixed in #176
